src/routes/raids/index.tsx

```diff
@@ -10,7 +10,7 @@ const AllRaids = () => {
   if (collectionData.state === 'success') {
     const data = collectionData.data
     return (
-      <>
+      <section role="main">
```

Comment:
Would it not make more sense to just use the `main` element?

Reply:
Yes